test/interceptors/request.test.ts

```diff
@@ -147,6 +147,60 @@ describe('test request interceptor', () => {
     // from revalidation
     expect(response2.cached).toBe(true);
     expect(response2.status).toBe(200);
+  });
+
+  it('tests last modified header handling with modified-since date', async () => {
+    const axios = mockAxios(
+      {},
+      { 'last-modified': 'Wed, 21 Oct 2015 07:28:00 GMT', 'cache-control': 'max-age=1' }
```

Comment:
same here, if you're using modifiedSince option, you should not need a last-modified header for the test to pass.